### DISCUSSION TITLE
Pandoc/markdown -> Zotero Live Citations with a citation style that uses footnotes

### DISCUSSION THREAD

**Discussion by sparkyc84** [retorquere/zotero-better-bibtex#2529] [Q&A]
Hi there,

Thanks @retorquere for the work on this plugin, it's great :-)  I've run into an issue and I'm not sure whether I'm doing something wrong or whether what I want is not possible.

I've got a markdown document with some citekeys in it.  I'm able to export this to DOCX with a csl that uses footnotes (for example, Chicago Manual of Style Full Note with Bibliograpy).

I'm also able to export it to Zotero live citations successfully.

When I do a Zotero live citation export then a zotero refresh, I'm prompted to pick a citation style as expected.  If I pick an in-text reference style (for example, Chicago Manual of Style Author-Date) everything works fine.

If I pick a style that uses footnotes though (for example, Chicago Manual of Style full note), the citations are not placed within a footnote and instead the reference appears within the text.

There is a workaround for this which involves picking a non-footnote citation style on the first refresh, then changing to a footnoted style in the Zotero Document Preferences panel within Word - but is there something I'm doing wrong that means it isn't working the first time around?

**Comment by retorquere** [comment]
I don't know, sorry - I think the zotero forums might be able to help, but I don't actually use this feature and have only a vague notion of how the word/LO stuff works. I reverse engineered the format and paste it in, if I'm doing that wrong, someone will have to tell me how to fix it.
